source/androidx.core/core/Transforms/Metadata.xml

```diff
@@ -99,5 +99,11 @@
         path="/api/package[@name='androidx.core.util']/class[@name='Preconditions']/method[@name='checkStringNotEmpty' and count(parameter)=3 and parameter[1][@type='T'] and parameter[2][@type='java.lang.String'] and parameter[3][@type='java.lang.Object...']]"
         />
 
+    <attr
```

Comment:
What is this for?

It causes an API breakage:

```
#### Removed Type Android.Support.V4.OS.IResultReceiver

#### New Type: Android.Support.V4.OS.ResultReceiverBase

```csharp
public interface ResultReceiverBase : Android.OS.IInterface, Android.Runtime.IJavaObject, Java.Interop.IJavaPeerable, System.IDisposable {
	// methods
	public virtual void Send (int resultCode, Android.OS.Bundle resultData);
}
```

It also appears to create misnamed nested types.  (Likely because it expects interfaces to start with a `I`.)

```
#### New Type: Android.Support.V4.OS.esultReceiverBaseConsts
#### New Type: Android.Support.V4.OS.esultReceiverBaseDefault
#### New Type: Android.Support.V4.OS.esultReceiverBaseStub
#### New Type: Android.Support.V4.OS.esultReceiverBase_Parcel
```
```

Reply:
Without that there are 2 errors:

```
./wsu-20220913/generated/androidx.core.core/obj/Release/monoandroid12.0/generated/src/Android.Support.V4.OS.IResultReceiver.cs(376,24): 
error CS0260: Missing partial modifier on declaration of type 'ResultReceiver'; another partial declaration of this type exists 
[./wsu-20220913/generated/androidx.core.core/androidx.core.core.csproj]
./wsu-20220913/generated/androidx.core.core/obj/Release/monoandroid12.0/generated/src/Android.Support.V4.OS.ResultReceiver.cs(9,3): 
error CS0579: Duplicate 'global::Android.Runtime.Register' attribute 
[/wsu-20220913/generated/androidx.core.core/androidx.core.core.csproj]
```

this seems to be culprit:

```
  <package name="android.support.v4.os" jni-name="android/support/v4/os">
    <interface abstract="true" deprecated="not deprecated" final="false" name="IResultReceiver" static="false" visibility="public" jni-signature="Landroid/support/v4/os/IResultReceiver;">
      <implements name="android.os.IInterface" name-generic-aware="android.os.IInterface" jni-type="Landroid/os/IInterface;" />
      <method abstract="true" deprecated="not deprecated" final="false" name="send" jni-signature="(ILandroid/os/Bundle;)V" bridge="false" native="false" return="void" jni-return="V" static="false" synchronized="false" synthetic="false" visibility="public">
        <parameter name="p0" type="int" jni-type="I" />
        <parameter name="p1" type="android.os.Bundle" jni-type="Landroid/os/Bundle;" />
        <exception name="RemoteException" type="android.os.RemoteException" />
      </method>
      <field deprecated="not deprecated" final="true" name="DESCRIPTOR" jni-signature="Ljava/lang/String;" static="true" transient="false" type="java.lang.String" type-generic-aware="java.lang.String" value="&quot;android.support.v4.os.IResultReceiver&quot;" visibility="public" volatile="false" />
    </interface>


    <class abstract="false" deprecated="not deprecated" extends="java.lang.Object" extends-generic-aware="java.lang.Object" jni-extends="Ljava/lang/Object;" final="false" name="ResultReceiver" static="false" visibility="public" jni-signature="Landroid/support/v4/os/ResultReceiver;">
      <implements name="android.os.Parcelable" name-generic-aware="android.os.Parcelable" jni-type="Landroid/os/Parcelable;" />
      <constructor deprecated="not deprecated" final="false" name="ResultReceiver" jni-signature="(Landroid/os/Handler;)V" bridge="false" static="false" type="android.support.v4.os.ResultReceiver" synthetic="false" visibility="public">
        <parameter name="handler" type="android.os.Handler" jni-type="Landroid/os/Handler;" />
      </constructor>
      <method abstract="false" deprecated="not deprecated" final="false" name="describeContents" jni-signature="()I" bridge="false" native="false" return="int" jni-return="I" static="false" synchronized="false" synthetic="false" visibility="public" />
      <method abstract="false" deprecated="not deprecated" final="false" name="onReceiveResult" jni-signature="(ILandroid/os/Bundle;)V" bridge="false" native="false" return="void" jni-return="V" static="false" synchronized="false" synthetic="false" visibility="protected">
        <parameter name="resultCode" type="int" jni-type="I" />
        <parameter name="resultData" type="android.os.Bundle" jni-type="Landroid/os/Bundle;" />
      </method>
      <method abstract="false" deprecated="not deprecated" final="false" name="send" jni-signature="(ILandroid/os/Bundle;)V" bridge="false" native="false" return="void" jni-return="V" static="false" synchronized="false" synthetic="false" visibility="public">
        <parameter name="resultCode" type="int" jni-type="I" />
        <parameter name="resultData" type="android.os.Bundle" jni-type="Landroid/os/Bundle;" />
      </method>
      <method abstract="false" deprecated="not deprecated" final="false" name="writeToParcel" jni-signature="(Landroid/os/Parcel;I)V" bridge="false" native="false" return="void" jni-return="V" static="false" synchronized="false" synthetic="false" visibility="public">
        <parameter name="out" type="android.os.Parcel" jni-type="Landroid/os/Parcel;" not-null="true" />
        <parameter name="flags" type="int" jni-type="I" />
      </method>
      <field deprecated="not deprecated" final="true" name="CREATOR" jni-signature="Landroid/os/Parcelable$Creator;" static="true" transient="false" type="android.os.Parcelable.Creator" type-generic-aware="android.os.Parcelable.Creator&lt;android.support.v4.os.ResultReceiver&gt;" visibility="public" volatile="false" />
    </class>
  </package>

```

which one to modify and how?